### PR TITLE
New package: hosts-gen-0.9

### PR DESCRIPTION
**Pull request by mustaqimM**

### CODE REVIEW

File: srcpkgs/hosts-gen/template [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+hostmakedepends="git"
+short_desc="Framework to generate /etc/hosts from various sources"
+maintainer="mustaqim <mustaqim+void@pm.me>"
+license="GPL-3.0-or-later"
+homepage="http://r-36.net/scm/hosts-gen"
+changelog="http://r-36.net/scm/hosts-gen/log.html"
+patch_args="-Np1"
+
+do_fetch() {
+	git clone git://r-36.net/hosts-gen ${pkgname}-${version}
```
Comment (maxice8) [inline srcpkgs/hosts-gen/template]: No, we need a release not a git package.

Comment (mustaqimM) [reply srcpkgs/hosts-gen/template]: The upstream repo allows for creating a release tarball with `make dist`, it doesn't provide one.

**Comment by maxice8** [comment]
On Tue, Feb 12, 2019 at 06:52:07AM +0000, Mustaqim Malim wrote:
> mustaqimM commented on this pull request.
> 
> 
> 
> > @@ -0,0 +1,21 @@
> +# Template file for 'hosts-gen'
> +pkgname=hosts-gen
> +version=0.9
> +revision=1
> +build_style=gnu-makefile
> +hostmakedepends="git"
> +short_desc="Framework to generate /etc/hosts from various sources"
> +maintainer="mustaqim <mustaqim+void@pm.me>"
> +license="GPL-3.0-or-later"
> +homepage="http://r-36.net/scm/hosts-gen"
> +changelog="http://r-36.net/scm/hosts-gen/log.html"
> +patch_args="-Np1"
> +
> +do_fetch() {
> +	git clone git://r-36.net/hosts-gen ${pkgname}-${version}
> 
> The upstream repo allows for creating a release tarball with `make dist`, it doesn't provide one.
> 
> -- 
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/pull/8567#discussion_r255822063

Sounds nice but we need a tagged release from which we can get a tarball

**Comment by Johnnynator** [comment]
Do we actually need a package for a 10 line shell script?
```sh
#!/bin/sh
 # See the LICENSE file for copyright and license details.
 
etcdir="/etc/hosts.d"
hostsfile="/etc/hosts"

hostfiles="$(ls -1 "${etcdir}" | grep '^[0-9]*-.*')"

 -e "$hostsfile" ] && cp "$hostsfile" "$hostsfile.bkp"
 
truncate -s 0 "${hostsfile}"
for f in $hostfiles;
do
	cat "${etcdir}/${f}" >> "${hostsfile}"
done

printf "%s file created.\n" "${hostsfile}"
````

**Pull request closed without merging**